spec/features/header_links_spec.rb

```diff
@@ -7,7 +7,7 @@
 
   context 'With MARC-based records' do
     before do
-      visit '/catalog/10585552'
+      visit '/catalog/99115031773506421'
```

Comment:
Question: Why is this not 9910585552?

Reply:
id: 99105855523506421 was not in my fixtures. The id I used is testing for the same features navbar, account dropdown, and record view, widgets, etc.